pkgs/build-support/build-fhs-userenv-bubblewrap/default.nix

```diff
@@ -126,6 +139,13 @@ let
       done
     fi
 
+    declare -a graphics_share
+    for dir in /run/opengl-driver/share/*; do
```

Comment:
Shouldn't we be merging in stuff from `/run/opengl-driver-32/share` as well?

Reply:
```suggestion
    for dir in /run/opengl-driver{,-32}/share/*; do
```

Reply:
https://github.com/NixOS/nixpkgs/pull/161576 seems to work just as well, I might prefer that, then rebase this.